### PR TITLE
Image Hosting VirtuDoc Symbol + Records Pages Controllers

### PR DESCRIPTION
**Pull request by spaceCowboy2071**
-Fixed image not appearing on certain navbars
-Added Controllers for the different records pages

### CODE REVIEW

File: src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/index.html [hunk 1]
```diff
@@ -14,7 +14,11 @@
     <nav class="navbar navbar-expand-lg bg-warning navbar-dark py-1">
         <div class="container-fluid">
           <div class="col-md-1 text-center d-none d-md-block">
+<<<<<<< Updated upstream:src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/index.html
```
Comment (ARMmaster17) [inline src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/index.html:17]: Looks like there's some leftovers from a merge conflict here on L17,19,21.

**Review by ARMmaster17 — APPROVED** [review]
Left a comment about merge conflict artifact. Looks like there's also some merge conflicts in the template files that need to be resolved. Otherwise LGTM.

File: src/main/java/com/virtudoc/web/controller/DoctorsRecordsController.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.virtudoc.web.controller;
+
+public class DoctorsRecordsController {
+    @GetMapping("/doctors_records")
```
Comment (ARMmaster17) [inline src/main/java/com/virtudoc/web/controller/DoctorsRecordsController.java:4]: Not a blocking issue, but wanted to open a discussion on this. Would it make more sense to combine the doctors and admins records pages into one? They have very similar content, and we can let the controller inject different data depending on who the currently logged-in user is.

**Review by ARMmaster17 — APPROVED** [review]

File: src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/index.html (hunk 1 above)
Comment (spaceCowboy2071) [reply src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/index.html]: They are removed now, I think the changes went through?

File: src/main/java/com/virtudoc/web/controller/DoctorsRecordsController.java (hunk 2 above)
Comment (spaceCowboy2071) [reply src/main/java/com/virtudoc/web/controller/DoctorsRecordsController.java:4]: That definitely sounds like an easier approach to me, maybe I could just have all three pages consolidate into one and code the back-end in a way that displays the page based on the logged in user's role?

**Comment by ARMmaster17** [comment]
Just as a heads-up, the static asset hosting issue was resolved in #135, so when you perform the merge conflict resolution, you can keep the `<img src=...` modifications on the `main` branch if everything looks like it works on your system.

On an unrelated note, can this PR be closed? It looks like your changes were included as part of commit 3611727bc63df53a228bec79b5cadc2f0eb45efb in #128. Feel free to close this PR if that's the case, or keep it open if there are other changes you would like to push that were not included.

**Pull request closed without merging**